src/transpileCodeblocks/compiler.ts

```diff
@@ -166,7 +190,7 @@ function createCompilerHost(
           );
     },
     resolveModuleNames(moduleNames, containingFile) {
-      return moduleNames.map((moduleName) => {
+      const mappedModules = moduleNames.map((moduleName) => {
```

Comment:
This could also stay a direct return and was only pulled apart for logging I think?